.github/workflows/rust-test.yml

```diff
@@ -49,6 +49,34 @@ jobs:
       - name: Test
         run: cargo test --all-features
 
+  coverage:
```

Comment:
❓ Can you save a run and just use the above, then append what's needed for Codecov?

Reply:
The above is a matrix. We could conditionally swap out half the commands but that seems somewhat annoying.